packages/store/addon/-private/system/relationships/state/has-many.ts

```diff
@@ -18,7 +19,7 @@ export default class ManyRelationship extends Relationship {
   key: string;
   constructor(
     store: any,
-    inverseKey: string,
+    inverseKey: string | null,
```

Comment:
this can never be null, only `Implicit` can be null which uses the base relationship class